Functional testcases failing with mount status & snapshot creation error

Docker plugin's Functional testcases on CentOS environment failing while unmounting & Snapshot creation.

Please find below common log for the same.

=======
Error logs
=======
hpe_3par_manager.py:310: in hpe_inspect_container_volume_mount
self.assertEqual(mount_status, True)
AssertionError: False != True

hpe_3par_manager.py:137: in hpe_create_snapshot
self.assertIn(snapshot_creation, volumes['Volumes'])
AssertionError: {'CreatedAt': '0001-01-01T00:00:00Z', 'Driver': 'dockerciuser/hpedockerplugin:test', 'Labels': None, 'Mountpoint': '', 'Name': 'docker_dc2389f1', 'Options': {'virtualCopyOf': 'docker_b92d06c2'}, 'Scope': 'global'} not found in [{'CreatedAt': '2018-08-20T22:01:54+05:30', 'Driver': 'local', 'Labels': None, 'Mountpoint': '/var/lib/docker/volumes/beeebe123272270ae7cc5917f2243c3744354e6aaac73e7aa75a037f9cdf347f/_data', 'Name': 'beeebe123272270ae7cc5917f2243c3744354e6aaac73e7aa75a037f9cdf347f', 'Options': None, 'Scope': 'local'}

CI job URL :- http://16.184.15.6:8080/job/docker-3par-iscsi-plugin-centos/156/consoleFull

--- a/tests/integration/hpe_3par_manager.py
+++ b/tests/integration/hpe_3par_manager.py
@@ -49,7 +49,11 @@ class HPE3ParVolumePluginTest(BaseAPIIntegrationTest):
     """
     This class covers all base methods to verify entities in Docker Engine.
     """
-    mount = r"hpedocker-(?:dm-uuid-mpath|dm-name-)"
+    # CentOS - hpedocker-dm-uuid-mpath-360002ac00000000000019e470001c00a
+    # Ubuntu - hpedocker-scsi-360002ac0000000000001d2750001c00a
+    # lets keep generic for now
+    mount = r"hpedocker-"
+    #mount = r"hpedocker-(?:dm-uuid-mpath|dm-name|scsi)"
     compile_mount_regex = re.compile(mount)
 
     def hpe_create_volume(self, name, driver, **kwargs):

